content/en/docs/concepts/workloads/controllers/cron-jobs.md

```diff
@@ -136,9 +136,14 @@ and set this flag to `false`. For example:
 
 ## {{% heading "whatsnext" %}}
 
-[Cron expression format](https://en.wikipedia.org/wiki/Cron)
-documents the format of CronJob `schedule` fields.
-
-For instructions on creating and working with cron jobs, and for an example of CronJob
-manifest, see [Running automated tasks with cron jobs](/docs/tasks/job/automated-tasks-with-cron-jobs).
-
+* Learn about [Pods](/docs/concepts/workloads/pods) and
```

Comment:
nit suggestion:
(My guess is that adding trailing slash will prevent the redirection caused by the rewrite feature of the web server.)

```suggestion
* Learn about [Pods](/docs/concepts/workloads/pods/) and
```

Reply:
Would also help with consistency since most other links end with a trailing slash.